crates/server/src/lib.rs

```diff
@@ -18,6 +18,7 @@ pub mod jsonrpc;
 #[cfg(feature = "admin")]
 mod middleware;
 mod verifysignature;
+mod verifysnsignature;
```

Comment:
these look similar, maybe we reorganize to the module structure `verify::starknet`?

Reply:
sgtm. 

even `verifysnsignature` should have full name `verifystarknetnsignature` -> we will forget "s", "p", "n", "k" parts pretty soon if we start using it in that way